packages/python/st2python.spec

```diff
@@ -6,12 +6,13 @@
 %if "%([ -x /usr/local/bin/python ] && echo -n 1)" == "1"
   %define __python /usr/local/bin/python
   %define __os_install_post() \
-      /usr/lib/rpm/redhat/brp-compress \
-      %{!?__debug_package:/usr/lib/rpm/redhat/brp-strip %{__strip}} \
-      /usr/lib/rpm/redhat/brp-strip-static-archive %{__strip} \
-      /usr/lib/rpm/redhat/brp-strip-comment-note %{__strip} %{__objdump} \
+      /usr/lib/rpm/brp-compress \
```

Comment:
It turns out I don't need to have EL8 vs EL < 8 logic here because the `brp` commands are in `/usr/lib/rpm/` as well as `/usr/lib/rpm/redhat/` on EL7. 

From a EL7 CentOS 7.7 install:
`[centos@ip-10-0-1-16 ~]$ cd /usr/lib/rpm
[centos@ip-10-0-1-16 rpm]$ ls
appdata.prov              check-files          find-debuginfo.sh  macros.php              perl.req           rpmdb_loadcvt   script.req
brp-compress              check-prereqs        find-lang.sh       macros.python           pkgconfigdeps.sh   rpmdb_recover   sepdebugcrcfix
brp-java-gcjcompile       check-rpaths         find-provides      mkinstalldirs           platform           rpmdb_stat      tcl.req
brp-python-bytecompile    check-rpaths-worker  find-requires      mono-find-provides      pythondeps.sh      rpmdb_upgrade   tgpg
brp-python-hardlink       config.guess         fontconfig.prov    mono-find-requires      pythondistdeps.py  rpmdb_verify
brp-strip                 config.sub           kabi.sh            ocaml-find-provides.sh  redhat             rpmdeps
brp-strip-comment-note    debugedit            libtooldeps.sh     ocaml-find-requires.sh  rpm2cpio.sh        rpm.log
brp-strip-shared          desktop-file.prov    macros             osgideps.pl             rpm.daily          rpmpopt-4.11.3
brp-strip-static-archive  elfdeps              macros.d           perldeps.pl             rpmdb_dump         rpmrc
check-buildroot           fileattrs            macros.perl        perl.prov               rpmdb_load         rpm.supp
[centos@ip-10-0-1-16 rpm]$ cd redhat/
[centos@ip-10-0-1-16 redhat]$ ls
brp-compress              brp-strip-comment-note    dist.sh                find-provides.pkgconfig  kmodtool                   rpmrc
brp-implant-ident-static  brp-strip-shared          find-provides          find-requires            libtool-handle-ppc64le.sh  rpmsort
brp-java-repack-jars      brp-strip-static-archive  find-provides.d        find-requires.ksyms      macros                     symset-table
brp-python-hardlink       config.guess              find-provides.ksyms    find-requires.libtool    redhat-hardened-cc1
brp-strip                 config.sub                find-provides.libtool  find-requires.pkgconfig  redhat-hardened-ld`